core/src/main/java/io/crate/metadata/IndexMappings.java

```diff
@@ -39,6 +39,15 @@
                         "org.elasticsearch.cluster.routing.DjbHashFunction", "Djb",
                         "org.elasticsearch.cluster.routing.Murmur3HashFunction", "Murmur3");
 
+    public static final Map<String, Object> DEFAULT_TABLE_MAPPING;
+
+    static {
+        DEFAULT_TABLE_MAPPING = new HashMap<>(1);
```

Comment:
what about:
```
    public static final Map<String, Object> DEFAULT_TABLE_MAPPING = MapBuilder.<String, Object>newMapBuilder()
        .put("_meta", new HashMap<String, Object>(1))
        .map();

    static {
        putDefaultSettingsToMeta(DEFAULT_TABLE_MAPPING.get("_meta"));
    }
```

this would at least remove the static metaMap definition